docs/source/installation.rst

```diff
@@ -14,20 +14,34 @@ To run the tests you can do::
   
 Requirements
 ============
-Ultimately, CLMM will depend on `CCL <https://github.com/LSSTDESC/CCL>`_, but until cluster_toolkit is incorporated into CCL, we have an explicit dependency. cluster_toolkit's installation instructions can be found `here <https://cluster-toolkit.readthedocs.io/en/latest/source/installation.html>`_.  The additional Python dependencies that you can get with pip are:
+
+To run the code, there are the following dependencies:
```

Comment:
We can mention "CLMM uses python 3" here and in `source/demo_pipeline` [link](https://github.com/LSSTDESC/CLMM/blob/issue/190/docs_revisions/docs/source/demo_pipeline.rst#setup)

Reply:
Added the python version in installation instrucitons!